go.mod

```diff
@@ -51,3 +51,5 @@ replace (
 )
 
 replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
+
+go 1.13
```

Comment:
Is that intentional?

Reply:
mh, no, not at all
thik go 1.13 adds it automatically, let me switch to go 1.12.x for the moemnt

Reply:
should be gone now